templates/kubevirt/kubevirt.yaml

```diff
@@ -82,7 +86,11 @@ spec:
       resourceName: virt-handler
       patch: '[{"op":"replace","path":"/spec/template/spec/tolerations","value":{{ $tolerationsAnyNode }}}]'
       type: json
-
+    # Add node affinity for the lowest virtualization level
+    - resourceType: DaemonSet
+      resourceName: virt-handler
+      patch: '[{"op":"add","path":"/spec/template/spec/affinity","value":{"nodeAffinity":{"requiredDuringSchedulingIgnoredDuringExecution":{"nodeSelectorTerms":[{"matchExpressions":[{"key":"node.deckhouse.io/dvp-nesting-level","operator":"In","values":["{{ $dvpNestingLevel }}"]}]},{"matchExpressions":[{"key":"node.deckhouse.io/dvp-nesting-level","operator":"DoesNotExist"}]}]}}}}]'
+      type: json
```

Comment:
Better use strategic. Also, will it work nicely with `nodeSelector` from https://github.com/deckhouse/virtualization/pull/1076 ?